lld/ELF/Driver.cpp

```diff
@@ -2976,6 +3026,9 @@ void LinkerDriver::link(opt::InputArgList &args) {
   // contain a hint to tweak linker's and loader's behaviors.
   config->andFeatures = getAndFeatures();
 
+  if (config->emachine == EM_AARCH64)
+    getAarch64PauthInfo();
```

Comment:
`AArch64` instead of `Aarch64`

Reply:
Thanks, fixed in 589c6455a929f41ed3a79fc7d91119586eb1ee7b